src/libxc/hyb_gga_xc_wb97.c

```diff
@@ -24,7 +24,7 @@ static const gga_xc_wb97_params par_wb97 = {
   { 1.00000e+00,  1.13116e+00, -2.74915e+00,  1.20900e+01, -5.71642e+00}, 
   { 1.00000e+00, -2.55352e+00,  1.18926e+01, -2.69452e+01,  1.70927e+01}, 
   { 1.00000e+00,  3.99051e+00, -1.70066e+01,  1.07292e+00,  8.88211e+00} 
-#if defined CUDA || defined CUDA_MPIV
```

Comment:
Why do we add an extra parameter for GPU code?

Reply:
These are the values of cam_omega. See my previous comment.

Reply:
Ah ok. Thank you for explaining.